doc/iterations/2024.1/shapes/add_support_for_libraries.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for libraries
+
+== Context
+
+Users want to have access to read only data (a.k.a libraries).
+
+== Key Result
+
+In the Details View, users could reference these read only data from their models' elements.
+
+== Solution
+
+1. When declaring templates, specifiers should be able to specify a list of libraries that will be accessible in the projects created from the templates.
+
+2. Specifiers should be able to associate libraries to projects' natures. Thus, when a project with a specific nature is created/uploaded, the libraries associated to this specific nature will be accessible.
+
+3. The `Blank project` has no nature and is not a template. So with the proposed solution, it will not be possible to use libraries in a `Blank project`. To handle this specific case of the `Blank project`, it should be possible to add libraries to every project without nature. This will be done programmatically, by the specifier.
+
+These libraries are models stored as files.
```

Comment:
We will have a lot to discuss with regard to the technical aspect of the solution. While I don't think the solution would be far from this, I would remove that to keep it in the ADR since there will be way more thinks to consider than just "stored as files"

Reply:
done